### ISSUE TITLE
Virtual Interfaces enabled/not?

### ISSUE THREAD

**Issue by TheMagicSkY** [aircrack-ng/rtl8188eus#40]
Hello,

This drivers support virtual interfaces (to pe set as AP)? I cannot get it work with Fluxion Captive Portal, please let me know thanks!

**Comment by woOzZ1** [comment]
That's what happened to me

**Comment by woOzZ1** [comment]
I have the same problem

**Comment by adhamkhaled-dev** [comment]
I don't think it is working. However, you can make deauth attack with your external adapter and fake AP with your built in adapter.

**Comment by ghost** [comment]
虚拟接口在监听模式下工作不能注入数据包成功发送出去。
